websites/A/AMC+/dist/metadata.json

```diff
@@ -7,13 +7,14 @@
 	"service": "AMC+",
 	"description": {
 		"en": "Award-winning shows and fan favorite movies – AMC+ is your destination for genre-defying storytelling. Ad Free and On Demand with new shows and movies added every week.",
-		"vi_VN": "Các chương trình đạt giải và các bộ phim yêu quý của người dùng - AMC+ là điểm đến của bạn nếu bạn muốn những câu truyện bất chấp thể loại. Không quảng các và theo nhu cầu các chương trình và bộ phim mới."
+		"vi_VN": "Các chương trình đạt giải và các bộ phim yêu quý của người dùng - AMC+ là điểm đến của bạn nếu bạn muốn những câu truyện bất chấp thể loại. Không quảng các và theo nhu cầu các chương trình và bộ phim mới.",
+		"nl": "Award winnende shows en fan favoriete films - AMC+ is uw bestemming voor genre-bepalende verhaal vertelling. Zonder advertenties en On Demand met elke week nieuwe shows en films."
```

Comment:
`Award winnende shows en fan favoriete films - AMC+ is uw bestemming voor genre-bepalende verhaal vertelling.`
@QkeleQ10 Hier zitten denk ik meerdere fouten in.

`Zonder advertenties en On Demand met elke week nieuwe shows en films.`
Misschien is dit beter:
`Advertentievrij en on-demand met elke week nieuwe shows en films.`